tools/question_converters/randexam/randexam2pl2.py

```diff
@@ -558,12 +558,12 @@ def check_library(library):
 
 
 ######################################################################
-def mkdir_or_die(dir):
-    if not os.path.exists(dir):
-        os.makedirs(dir)
+def mkdir_or_die(directory):
+    if not os.path.exists(directory):
+        os.makedirs(directory)
```

Comment:
Weird, GitHub seems to have lost the comment I left on this in my review. Couldn't we replace this whole function with `os.makedirs(..., exist_ok=True)`? I know this is unrelated to this PR, feel free to ignore.

Reply:
I would rather consider blanket-applying those sort of changes, perhaps a part of https://docs.astral.sh/ruff/rules/#flake8-use-pathlib-pth, rather than in this PR.